runner/runners/queue.go

```diff
@@ -24,7 +26,11 @@ func NewQueueRunner(
 	statusManager := NewStatusManager()
 	inv := NewInvoker(exec, filer, output, tmp)
 	controller := &QueueController{statusManager: statusManager, inv: inv, capacity: capacity}
-	return &Service{controller, statusManager, statusManager}
+	runner := &Service{controller, statusManager, statusManager}
+	log.Println("************** runner definition")
```

Comment:
I'd try just one of these blocks located in ice/eval.go which prints the values map. If that works we can replace the other blocks that are sprinkled throughout.